docs/CONTRIBUTING.MD

```diff
@@ -4,83 +4,84 @@ Render Markdown in VS Code
 
-#### generate-component.sh
+## Creating a New Component
 
-This script creates a new component by referencing a `TemplateComponent` in the project root. Upon running, the user will be prompted to provide a name for the new component. The script will then create a directory with a basic working component and Storybook story.
+All of the components live in `src/components/`, and each new component should be included here. Components can be created from scratch, but we recommended that you use an existing component as a reference. Additionally, you may generate a simple boilerplate component using `npm run generate-component`.
+
+The `generate-component.sh` script creates a new component by referencing the `TemplateComponent` directory in the project root. Upon running, the script will prompt you to provide a name for the new component and create a directory with a basic working component and Storybook story.
 
 Example:
 
 ```bash
-> npm run generate-component
+$ npm run generate-component
 
 This script generates a new template component with a specified name.
-Enter the new component name: Button
+Enter the new component name: NewComponent
 
 # Output:
-# src/components/Button/
-# ├── Button.scss
-# ├── Button.stories.tsx
-# ├── Button.test.tsx
-# └── Button.tsx
+# src/components/NewComponent/
+# ├── NewComponent.scss
+# ├── NewComponent.stories.tsx
+# ├── NewComponent.test.tsx
+# └── NewComponent.tsx
 ```
 
-At the present time, this script does not add imports for your new component in `_uswds-theme-custom-styles.scss` or in `index.tsx`. These are required if you want to include custom CSS or include your component in the packaged version of this code.
+At the present time, this script does not add imports for your new component in `_uswds-theme-custom-styles.scss` or in `index.tsx`. These are required if you want to include custom CSS or include your component in the packaged version of this code. See [Using Custom Styles](#using-custom-styles) for more.
 
-### Pull Requests
+Finally, add your new component to `src/index.tsx` in order to appear in Storybook.
 
-We use pull request to drive a variety of actions and workflows in the project
+## Pull Requests
 
-Once you are satisfied with the code you have made it is time to make a pull request. Be sure to follow the pull-request template when creating your pull request. Upon doing this and provided all of the additional checks pass your code will be reviewed and approved if the changes are in line with the projects plans.
+We use pull request to drive a variety of actions and workflows in the project.
 
-### Testing
+Be sure to follow the template when creating your [pull request on GitHub](https://github.com/CMSgov/macpro-ux-lib/pulls). Upon doing this, and provided all of the additional checks pass, your code will be reviewed and approved if the changes are in line with the projects plans.
 
```

Comment:
# Deployment

The short version of the CICD Pipeline is `Pull Request → Github Actions → AWS Amplify Temporary Environments → Accessible Application → Pull Request Approval → Master deployment & New Version `

All of the deployments start with new code on a branch. A branch environment is created and this is where automated and manual testing occurs. Once all of the checks have been passed on a branch, it is pulled into the Master branch. The master branch specifically lives in the git-hub actions branch `gha-pages` [LINK](https://cmsgov.github.io/macpro-ux-lib) and it serves as our most up to date deployment environment.  and it is where live users have access to the new features and new components.

Reply:
Excellent addition!